app/models/notification.rb

```diff
@@ -4,4 +4,22 @@ class Notification < ActiveRecord::Base
   belongs_to :user
 
   validates_presence_of :info_request_event, :user, :frequency, :send_after
+
+  before_validation :calculate_send_after
+
+  INSTANTLY = 'instantly' # Notify me about this as soon as possible
+  DAILY = 'daily' # Notify me about this at most once a day
```

Comment:
I chose a string because the db stores it the `frequency` as a string and I didn't want to have to remember to `to_sym` or `to_s` it everywhere we need to compare the two. Is there a trick I'm missing to make that painless though (without using a gem like symbolize?).

Ah - I guess I should use http://api.rubyonrails.org/v4.1/classes/ActiveRecord/Enum.html now that we're on 4.1